macros/anaUltraThreshold.py

```diff
@@ -244,6 +244,7 @@
 r.gStyle.SetOptStat(0)
 print 'Saving vt1Max distributions'
 for vfat in range(0,24):
+    if (vfat_mask >> vfat) & 0x1: continue
     r.gStyle.SetOptStat(0)
     canv_vt1Max.cd(vfat+1)
     dict_hMaxVT1[vfat].Draw("hist")
```

Comment:
Not knowing much of anything about this code, isn't it better to simply do something like (starting from line 246):
```python
for vfat in dict_hMaxVT1.keys():
    r.gStyle.SetOptStat(0)
    canv_vt1Max.cd(vfat+1)
    dict_hMaxVT1[vfat].Draw("hist")
...
```

If the code crashes when some data from some VFAT is missing, this is the better way to do the analysis, and should (probably) be changed in many places